pkg/acceptance/bettertestspoc/config/model/database_role_model_gen.go

```diff
@@ -43,6 +46,22 @@ func DatabaseRoleWithDefaultMeta(
 	return d
 }
 
+func (r *DatabaseRoleModel) ToConfigVariables() tfconfig.Variables {
```

Comment:
We should not generate this method every time. Let's just have a generic function expecting the ResourceModel

Reply:
👍 Placed it next to FromModel function in config package